phylogenetic/rules/merge_sequences_usvi.smk

```diff
@@ -21,22 +21,141 @@ This part of the workflow usually includes the following steps:
 
 """
 
-rule append_usvi:
-    """Appending USVI sequences"""
-    input:
-        sequences = "data/sequences.fasta",
-        metadata = "data/metadata.tsv",
-        usvi_sequences = "data/sequences_usvi.fasta",
-        usvi_metadata = "data/metadata_usvi.tsv"
+# ------------- helper functions to collect, merge & download input files ------------------- #
```

Comment:
Great idea to separate this out into a separate snakemake rules file. I should have done this for avian-flu!

I suggest we drop all the references to "USVI" - e.g. in the opening comment of this file & the filename itself. We'll use it for USVI-data no doubt, but that's a config setting.

Reply:
Just a heads-up, I pushed an initial rebase on a new branch [refactor-additional-inputs-rebase](https://github.com/nextstrain/zika/compare/main...refactor-additional-inputs-rebase). I plan to hard rebase this branch to that branch if I don't hear any major push back next week. To demonstrate that nextstrain run still can find the USVI workflow data, can run:

```
nextstrain setup zika@refactor-additional-inputs-rebase
mkdir out
nextstrain run zika@refactor-additional-inputs-rebase out
nextstrain view out/auspice
```

I'll continue adding the further fixups to documentation and naming to that branch. Also feel free to push changes.

Reply:
Please feel free to rebase as you wish - I'll pick up from wherever this GitHub branch points next week. 

Re: the search order for local (input) file paths - that's a great question, and I don't think I fully covered these cases in avian-flu. I think right now avian-flu only searches relative to the analysis directory. I wonder what the implications would be of treating local inputs akin to a config-referenced file, i.e. using the same path resolution algorithm.

Reply:
Rebased, thanks! And dropped usvi from smk name in https://github.com/nextstrain/zika/pull/80/commits/5a014cbc75f2eb1df8b2e89d5c180ee66ed4f510

Reply:
Agree, fixed in https://github.com/nextstrain/zika/pull/80/commits/f8634a1cf8ebc8da43f2a59e3aee9faa22a495eb